### PR TITLE
AAD rename - gRPC node topics

### LINKED ISSUE #30590: ADD rename for gRPC docs
Contributes to #30185

In all gRPC docs (and any includes they use)

Replace "Azure Active Directory," "Azure AD," and "AAD"

List topics here:

- [x] grpc/authn-and-authz.md

Examples:

**Product references:**
Azure Active Directory (AAD)
  becomes
Microsoft Entra (ME-ID)

Azure AD
  becomes
Microsoft Entra ID

AAD
  becomes
ME-ID

**Topic file names and their links:**
azure-active-directory.md
  might become
microsoft-entra-id

**Links:**
[Azure Active Directory (AAD)](xref:blazor/security/webassembly/hosted-with-azure-active-directory)
  becomes
[Microsoft Entra ID (ME-ID)](xref:blazor/security/webassembly/hosted-with-microsoft-entra-id)

### PR DESCRIPTION
**Pull request by wadepickett**
Fixes #30590

Azure Active Directory rename update for all gRPC topics in the gRPC node.
Criteria: replace "Azure Active Directory," "Azure AD," and "AAD" and looked for any links to docs with those names in the filename or title.

There was just one topic that needed an update in that gRPC node:  authn-and-authz.md

See related issues #30591 and #30592

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/grpc/authn-and-authz.md](https://github.com/dotnet/AspNetCore.Docs/blob/210f27d2f27e488dea2452b55084f0da4e8e2556/aspnetcore/grpc/authn-and-authz.md) | [Authentication and authorization in gRPC for ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/grpc/authn-and-authz?branch=pr-en-us-30593) |

### CODE REVIEW

**Review by tdykstra — COMMENTED** [review]
Why isn't this just Microsoft Entra ID (why is ME-ID in parens used)

File: aspnetcore/grpc/authn-and-authz.md [hunk 1]
```diff
@@ -207,7 +207,7 @@ public Ticketer.TicketerClient CreateClientWithCert(
 
 Many ASP.NET Core supported authentication mechanisms work with gRPC:
 
-* Azure Active Directory
+* Microsoft Entra (ME-ID)
```
Comment (wadepickett) [inline aspnetcore/grpc/authn-and-authz.md:210]:
```suggestion
* Microsoft Entra ID
```

File: aspnetcore/grpc/authn-and-authz.md [hunk 2]
```diff
@@ -476,7 +476,7 @@ public Ticketer.TicketerClient CreateClientWithCert(
...
```
Comment (wadepickett) [inline aspnetcore/grpc/authn-and-authz.md:479]:
```suggestion
* Microsoft Entra ID
```

**Comment by wadepickett** [comment]
@tdykstra, you are right, it should have been "Microsoft Entra ID".   I have changed it, thanks!

**Review by tdykstra — APPROVED** [review]

**Pull request merged**